File: Week 2/Contest (Mar 20)/ABC/elijah.cs
using System;
using System.Linq;

class Entry
{
    static void Main()
    {
        int[] nums = Console.ReadLine()
            .Split(' ')
            .Select(s => int.Parse(s))
            .ToArray();
            
        Array.Sort(nums);
            
        char[] chars = Console.ReadLine().ToCharArray();
        
        foreach (char c in chars)
        {
            if (c == 'A')
            {
                Console.Write(nums[0].ToString() + " ");
            }
            else if (c == 'B')
            {
                Console.Write(nums[1].ToString() + " ");
            }
            else if (c == 'C')
            {
                Console.Write(nums[2].ToString() + " ");
            }
        }
    }
}

File: Week 2/Extra Practice Problems (Week 2)/2592: Maximize Greatness of an Array/elijah.cpp
class Solution {
public:
    int maximizeGreatness(vector<int>& nums)
    {
        sort(nums.begin(), nums.end());

        int result = 0;
        for (int i = nums.size() - 2, j = nums.size() - 1; i >= 0; i--)
        {
            if (nums[i] < nums[j])
            {
                j--;
                result++;
            }
        }

        return result;
    }
};
